WIP: Add exemplar support tasks (#408)
alphagov/paas-team-manual: source/guides/common_support_tasks.html.md

- if the email account belonging to the absent org manager can be made active again to send an email authorising the change.

We will also email every user currently in the organisation to explain what has been done.

# Common Support questions

> ## Which Databases do you support?

Reviewer: ```suggestion
## Which databases do you support?
```

> - Postgres

Reviewer: ```suggestion
- Postgres (RDS)
```

> - Postgres (RDS)

Reviewer: ```suggestion
- [Postgres](https://admin.cloud.service.gov.uk/marketplace/efadb775-58c4-4e17-8087-6d0f4febc489) (RDS)
```

> - MySQL

Reviewer: ```suggestion
- MySQL (RDS)
```

> - MySQL (RDS)

Reviewer: ```suggestion
- [MySQL](https://admin.cloud.service.gov.uk/marketplace/8ffb13cb-705f-4cee-9ffd-5a950f9f3048) (RDS)
```

> - Elasticsearch

Reviewer: ```suggestion
- [Elasticsearch](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
```

- Redis

## Some plans in the marketplace are displaying as free. What does this mean?

> This is a horrible bit of Cloud Foundry UI: "free" really means "free for 3 months while you are in trial, and then you need to pay."

Reviewer: ```suggestion
This is a horrible bit of Cloud Foundry user interface: "free" really means "free for 3 months while you are in trial, and then you need to pay."
```

> - Redis

Reviewer: ```suggestion
- [Redis](https://admin.cloud.service.gov.uk/marketplace/0e4b6fb6-243c-4024-8604-41635d1233cc)
```

> 

Reviewer: ```suggestion
- [Influxdb](https://admin.cloud.service.gov.uk/marketplace/0b7da7a9-2fd7-4dca-9e25-0cf8eddd0f88)

```

## Can I get direct access to the RDS database instances?

No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 

## What is your Elasticsearch offering?

> We currently offer Elasticsearch 7 

Reviewer: ```suggestion
We currently offer [Elasticsearch 7](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
```

> We currently offer Elasticsearch in private beta for v5 and v6

Reviewer: ```suggestion
We currently offer Elasticsearch 7 
```

> ## When will your Elasticsearch reach public beta?

Reviewer: ```suggestion
## What is the status of the Elasticsearch backing service?
```

> We are currently in private beta and are currently testing the new service with tenants. We are confident that we have a reliable service and are working to solicit feedback from beta user before making the appliation more widely available.

Reviewer: ```suggestion
GOV.UK PaaS is [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment)
```

- if the email account belonging to the absent org manager can be made active again to send an email authorising the change.

We will also email every user currently in the organisation to explain what has been done.

# Common Support questions

> ## Which Databases do you support?

Reviewer 2: I would have databases in lowercase here.

- Postgres
- MySQL
- Elasticsearch
- Redis

> ## Some plans in the marketplace are displaying as free, what does this mean?

Reviewer 2: ```suggestion
## Some plans in the marketplace are displaying as free. What does this mean?
```

> This is a horrible bit of CF UI: "free" really means "free for 3 months while you are in trial, paid after"

Reviewer 2: ```suggestion
This is a horrible bit of Cloud Foundry UI: "free" really means "free for 3 months while you are in trial, and then you need to pay."
```

## Can I get direct access to the RDS database instances?

> No, the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 

Reviewer 2: ```suggestion
No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
```

> ## What is your elasticsearch offering?

Reviewer 2: ```suggestion
## What is your Elasticsearch offering?
```

> We currently offer elasticsearch in private beta for v5 and v6

Reviewer 2: ```suggestion
We currently offer Elasticsearch in private beta for v5 and v6
```

> ## When will your elasticsearch reach public beta?

Reviewer 2: ```suggestion
## When will your Elasticsearch reach public beta?
```

> We are currently in private beta and are currently testing the new service with tenants. We are confident that we have a reliable service and are working to solicit feedback from beta user before making the appliation more widely available.

Reviewer 2: ```suggestion
We are currently in private beta and are currently testing the new service with tenants. We are confident that we have a reliable service and are working to solicit feedback from beta users before making the application more widely available.
```

Reviewer: PaaS is now a [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment) so we should remove any mention of private beta

> GOV.UK PaaS is [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment)

Reviewer: ```suggestion
GOV.UK PaaS is a [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment)
```
